### PR TITLE
feat(helm-charts): Add sqlPort config to enable SQL Api

### PR DESCRIPTION
**Pull request by lvauvillier**
- Add `config.sqlPort` config to enable SQL Api (disabled by default)
- Remove image.tag v* restriction.
- Lint

### CODE REVIEW

**Comment by rpaik** [comment]
@adnanrahic can you help with the review?

File: examples/helm-charts/cubejs/values.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ config:
   ##
   apiPort: 4000
 
+  ## The port for a Cube.js deployment to listen to SQL connections on
+  ##
+  sqlPort:
```
Comment (adnanrahic) [inline examples/helm-charts/cubejs/values.yaml:38]: Do you think it's useful to add the default SQL port here? Similar to how you added the default API port.
I'm just asking. If you think it is not needed, let's skip it.

**Review by adnanrahic — APPROVED** [review]
Thanks so much for this @lvauvillier ! Looks good, we can merge it once we decide on if we need a default value for the `sqlPort`. :smile:

File: examples/helm-charts/cubejs/values.yaml (hunk 1 above)
Comment (lvauvillier) [reply examples/helm-charts/cubejs/values.yaml:38]: According to the doc: https://cube.dev/docs/backend/sql, the sql API is enabled only if we set a port.

I reflected here this behavior: no value disabled, value enabled. The API is disabled by default.

**Pull request merged**